submit.go

```diff
@@ -291,6 +292,13 @@ func parseRequest(w http.ResponseWriter, req *http.Request, reportDir string) *p
 	return p
 }
 
+var uaParser *uaparser.Parser = uaparser.NewFromSaved()
```

Comment:
https://pkg.go.dev/github.com/ua-parser/uap-go/uaparser#NewFromSaved is sadly lacking. I guess it loads from a preset list of user-agents.